indico/web/templates/forms/radio_buttons_widget.html

```diff
@@ -3,12 +3,33 @@
 {% set vertical_alignment = field.option_orientation == 'vertical' %}
 
 {% block html %}
-    <div class="{{ 'i-group' if vertical_alignment else 'group i-selection' }}">
+    <div id="{{ field.id }}" class="{{ 'i-group' if vertical_alignment else 'group i-selection' }}">
         {% for subfield in field %}
             <div class="{{ 'i-radio' if vertical_alignment else 'inline-vcentered'}} radio-item-{{ loop.index0 }}">
                 {{ subfield(**input_args) }}
                 {{ subfield.label(class_=('i-button' if not vertical_alignment)) }}
             </div>
         {% endfor %}
     </div>
+    <script>
```

Comment:
widget templates have a `javascript` jinja block

Reply:
yeah, but then we should also remember about moving JS code from existing templates to this block (e.g. switch_widget)